src/QirTools/qirlib/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "qirlib"
+version = "0.1.0"
+edition = "2018"
+description = "QIR generation library for use with the Quantum Development Kit."
```

Comment:
It may be helpful to note in the description that this is currently limited to Base Profile / nonadaptive programs (informally, "circuits").